user-guides/ivche1337/Streams-And-Topics-Part2.md

```diff
@@ -0,0 +1,47 @@
+# Streams & topics, Part 2
+
+## Unsubscribe from a stream
+- Unsubscribing from a stream through the Streams page
+- Unsubscribing from a stream through the stream sidebar
+
+* No typos
+* Steps clear and descriptive
+* Screenshots would ease the process
+* I followed the steps easy
+## Change the privacy of a stream
+* No typos
+* Steps clear and descriptive
+* Screenshot would ease the process
+* Begginer friendly
+* I followed the steps easy
+
+## Organize the Streams sidebar
+- Pinning
+- Searching
+- Muting
```

Comment:
Can you explain what this means?

Reply:
Those are the features regarding the sidebar, pin, mute, search..

Reply:
Ok I guess you don't need to write them out.